### PR TITLE
tests:Add doc validation to travis CI

### PR DESCRIPTION
**Pull request by tobine**
Run doc validation following build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,7 +46,8 @@ script:
   - if [[ "$VULKAN_BUILD_TARGET" == "LINUX" ]]; then ./update_external_sources.sh; fi
   - if [[ "$VULKAN_BUILD_TARGET" == "LINUX" ]]; then cmake -H. -Bdbuild -DCMAKE_BUILD_TYPE=Debug; fi
   - if [[ "$VULKAN_BUILD_TARGET" == "LINUX" ]]; then make -C dbuild -j $core_count; fi
-  - if [[ "$VULKAN_BUILD_TARGET" == "LINUX" ]]; then LD_LIBRARY_PATH=dbuild/loader:$LD_LIBRARY_PATH VK_LAYER_PATH=dbuild/layers VK_ICD_FILENAMES=dbuild/icd/VkICD_mock_icd.json dbuild/tests/vk_layer_validation_tests; fi
+  - if [[ "$VULKAN_BUILD_TARGET" == "LINUX" ]]; then pushd dbuild/tests;./vkvalidatelayerdoc.sh; fi
```
Comment (cnorthrop) [inline .travis.yml]: Any reason not to have the popd on this line?  Would look a bit cleaner.

Comment (tobine) [reply .travis.yml]: The last thing run on the line determines pass/fail so when popd is the last thing it masks any doc validation failure.

Comment (jzulauf-lunarg) [reply .travis.yml]: I'm generally not a fan of changing the directory during a linked series of step.  what if one were to run the chdir'ing command in a ( ... ) subshell?

( cd dbuild/test; ./vkvalidatelayerdoc.sh); fi

I just tested and the last exit status of the sub-shell is was is current after returning from the subshell.

Comment (cnorthrop) [reply .travis.yml]: Switching to a subshell looks good to me.

**Pull request merged**